### ISSUE TITLE
My blog became nonresponsive to mobile phones

### ISSUE THREAD

**Issue by AutomationMan808** [danpros/htmly#517]
Any suggestions?

**Comment by Hjertesvikt** [comment]
To make suggestions it would be nice to have more information.
What modification did you make to your theme?
Did you remove any javascript library?

**Comment by AutomationMan808** [comment]
No I did not
As far as changes nothing I hadn't done to my other HTMLy blogs. Add some iframes to run programs a Twitter feed and a feed along the side. 
https://increasesales.info/

Okay that's weird. When I narrow the page on the computer it's responsive. On my phone it's not. I tried on both my Google browser and downloaded Opera and tried in on there. On both browsers on my phone my site was unresponsive

Rick

**Comment by Hjertesvikt** [comment]
I can't find a bootstrap library in your home page?

**Comment by Hjertesvikt** [comment]
Rick,
Did you look at your web page with Javascript console (Best is Chrome, but Firefox is OK).
There are several errors in your site.
Jean-Pierre

**Comment by Hjertesvikt** [comment]
For comparison, if you look at HTMLy, there is no error reported by the Javascript console

**Comment by AutomationMan808** [comment]
Where are those files located I should be able to copy them from another installation 

Thank you 
Rick

**Comment by Hjertesvikt** [comment]
I guess it's not simply a matter to add bootstrap to your pages, you have to match the "class" statements in HTML code. with the Javascript code.

**Comment by AutomationMan808** [comment]
Sounds like I might need to reinstall then upload my posts

**Comment by Hjertesvikt** [comment]
And if you simply try another theme, just to check if it changes something. It would be a quick check.

**Comment by AutomationMan808** [comment]
I did try another theme. Tried a different browser on my phone and then tried my son's phone. All had the same results. Still curious as to why it was responsive when I narrowed the computer's browser but not responsive on the phone. Well, like you said,. There were errors. 

Rick

**Comment by AutomationMan808** [comment]
You gotta love HTMLy and flat file systems in general. I uploaded the most recent version of HTMLy. I did a backup even though I'm fanatical about doing a backup of all my sites using Filezilla and even that syncs with Google Drive (computer tutor and troubleshooter for a living) but, I didn't know what to expect.

Since the new upload didn't overwrite any content files created everything was in there. Even my login. Now I'll do a full clean backup and upload my layout and system files. I'll test each time to see if something breaks. If it does I'll reverse the action with the clean latest install.

Thanks a lot for your help,

Rick
